grails-app/controllers/imod/AssessmentTechniqueController.groovy

```diff
@@ -167,15 +171,16 @@ class AssessmentTechniqueController {
 		// persist new technique to database
 		newTechnique.save()
 
-// This checks when a technique is assigned to a learning objective
 
 		if(params.assigncheck == true) {
 		// get current user object

```

Comment:
The code inside the `if` block.
between the `{}` should be tabbed one additional level.